src/cmd/ksh93/tests/emacs.exp

```diff
@@ -0,0 +1,279 @@
+# Check emacs keybindings
+set pid [spawn $ksh]
+expect_prompt
+
+send "echo hello\r"
```

Comment:
You can drop this test. I only had it in mine because it was the very first such test we've written. We should probably move it, and any other test that does nothing more than establish baseline behavior, into a separate unit test whose name ensures it is run before any other interactive test.

Reply:
It just acts as sample data for testing `ctrl-n` and `ctrl-p` bindings. I can drop the line below it however.

Reply:
Ahh, yes, I see. Okay, just put add a comment that this is to inject a command in the history. Keep the `expect_prompt`.